config/webpack/utils/startConfig.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+
+const startConfigPath = path.join(__dirname, '../../render/startConfig.json');
+
+const writeStartConfig = (environment, site) => {
+  fs.writeFileSync(
+    startConfigPath,
+    JSON.stringify({
+      environment,
+      site
+    })
```

Comment:
Save prettier catching this, maybe stringify with indentation, eg. `JSON.stringify(..., null, 2)`